### PR TITLE
[13.0][ADD] account_purchase_stock_report_non_billed: New module

### PR DESCRIPTION
**Pull request by CarlosRoca13**
Module that extends account_sale_stock_report_non_billed to add the non billed stock moves comming from purchase orders.

This PR depends on:

- [x] https://github.com/OCA/account-financial-reporting/pull/896

cc @Tecnativa TT38550

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: account_purchase_stock_report_non_billed/README.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--financial--reporting-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-financial-reporting/tree/15.0/account_purchase_stock_report_non_billed
+    :alt: OCA/account-financial-reporting
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-financial-reporting-15-0/account-financial-reporting-15-0-account_purchase_stock_report_non_billed
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/91/15.0
```
Comment (sergio-teruel) [inline account_purchase_stock_report_non_billed/README.rst]: 15.0 or 13.0?

**Review by sergio-teruel — APPROVED** [review]

File: account_purchase_stock_report_non_billed/README.rst (hunk 1 above)
Comment (CarlosRoca13) [reply account_purchase_stock_report_non_billed/README.rst]: Changed

**Comment by CarlosRoca13** [comment]
Y add a fix on `account_sale_stock_report_non_billed` and tests for the use case mentioned on  `account_sale_stock_report_non_billed` and `account_purchase_stock_report_non_billed` both modules

File: account_sale_stock_report_non_billed/models/stock_move.py [hunk 2]
```diff
@@ -53,6 +53,12 @@ def check_is_return(self):
     def get_total_devolution_moves(self):
         total_qty = 0
         for move in self:
+            # Avoid moves related to returns that not update qty on stock
+            if (
+                any(x.origin_returned_move_id for x in move.picking_id.move_lines)
```
Comment (pedrobaeza) [inline account_sale_stock_report_non_billed/models/stock_move.py]: Why not `if move.origin_returned_move_id and not move.to_refund`? Each move can have a different selection.

Comment (CarlosRoca13) [reply account_sale_stock_report_non_billed/models/stock_move.py]: Yeah, you are right

Changed

**Pull request merged**